### ISSUE TITLE
Make auto update cron job optional at install time

### ISSUE THREAD

**Issue by oblivian** [netdata/netdata#2932]
Hello,

Thanks for a fantastic tool. Works great!
One thing suggestion though, the default Linux installer installs (kind of silently, or at least you need to pay close attention) a bash/shell update script as a daily cron job under the root account. Don't you think that is extremely dangerous? In theory it means that thousands of servers are just one hack of your GitHub account from being completely wiped or hijacked. I can of course delete the cron job, and I do. I just think it would be easier if that was an option in the installer to enable or not the auto updater, rather than having to go back and delete the cron jobs.

Thanks.

**Comment by ktsaou** [comment]
Hi,

This is done by the kickstart script. If you clone the repo and run the installer by hand, it does not do this by default (you will need to enable it). The kickstart also supports the command line option `--no-updates` to disable auto-updates.

When I took this decision, I researched what mainstream commercial solutions do. I found out that most commercial monitoring solutions, auto-update by default.

To decide, I also evaluated 2 scenarios:

1. What happens if my github account is hacked and people auto-update?
2. What happens if we find a major issue in netdata and people do not auto-update?

Which one is the most likely to happen? Which one would be the harder to handle?

Ideally, I would prefer to disable auto-update by default, if I had the means to update everyone using netdata about important security issues. But this is not the case. I don't have any means to update them.

So, I thought that there are 2 use cases:

1. people, like you, who know what they are doing, watch this repo and check how they will update netdata. These people can disable auto-update.

2. people who don't actually understand all the implications, and just install netdata.

I think a default kickstart installation is for the later. These people will stay vulnerable forever if we find an issue with netdata and I will have absolutely no means to let them know.

**So, my decision to enable auto-updates for default kickstart installations, is to actually protect people that are not mature sysadmins.**

---

Ideally this whole process should be handled by distros. Unfortunately, most distros are hopeless. They have lost the race of applications packaging, long time ago.
